simulate.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import (absolute_import, division,
+                        print_function, unicode_literals)
+
+import numpy as np
+import scipy as sp
+from scipy import signal
+import h5py
+import math
+import PSS_utils as utils
+import sys
+sys.path.append('/Users/nearclouding/Desktop/UTGRV_REU/nanograv_group')
+import VersionZeroPointZero as PSS
+
+#Initializing the signal
+S1=PSS.signal(Nt=2000)
+
+class Simulate():
+    """Class that simulates the pulsar signal
+       Insert more information about the class
+    """
+    def __init__(self, signal_class,pulsar_class,ism_class):
+        self.signal_class=signal_class
+        self.pulsar_class=pulsar_class
+        self.ism_class=ism_class
+        #NOTE need to give signal_class, pulsar_class, ism_class attributes?
+        #What else goes in here?
+
+    #Make the pules from initialized signal
+    def make_the_pulses():
+        self.P1=self.PSS.Pulsar(S1)
```

Comment:
We definitely want to use the functions that we've already written, so the latter is correct. So as we have written the code you would just call the functions in exactly the way you cite above. 

That said, there could be a way to right the code where someone only puts in the `signal_class` and then the code takes care of the rest. The `signal_class.MetaData` probably will have all of the information needed to do the calculations. I think for now, let's stick with this plan, but it's probably something to keep in mind as we move forward.